python/ray/actor.py

```diff
@@ -841,7 +841,10 @@ def __getattribute__(self, attr):
         return object.__getattribute__(self, attr)
 
     def __repr__(self):
-        return "Actor(" + self._ray_actor_id.hex() + ")"
+        return "Actor({})".format(self._ray_actor_id)
+
+    def __str__(self):
+        return "RayActor({})".format(self._ray_class_name)
```

Comment:
Why put different information in these two functions? I think they should be the same.

Reply:
Hm, not totally sure - I read this particular comment https://stackoverflow.com/a/1436756/2213289 which seems to suggest you want the unique identifier in the repr.